apps/cookbook/src/app/showcase/stock-chart-showcase/stock-chart-showcase.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="example">
-  <cookbook-stock-chart-example></cookbook-stock-chart-example>
+  <cookbook-stock-chart-example #example></cookbook-stock-chart-example>
+  <cookbook-code-viewer [html]="example.template"></cookbook-code-viewer>
```

Comment:
Please align with existing showcases:
```suggestion
<cookbook-example-viewer [html]="example.template">
  <cookbook-stock-chart-example #example></cookbook-stock-chart-example>
</cookbook-example-viewer>
```

Reply:
Done.